### PR TITLE
Add analytics components to store

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/apis/stats-api.jag [hunk 1]
```diff
@@ -0,0 +1,281 @@
+<%
```
Comment (DilanUA) [inline modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/apis/stats-api.jag:1]: are these implementations still in use?

File: modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/app.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/*
```
Comment (DilanUA) [inline modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/app.js:1]: are these implementations still in use?

File: modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/pages/device-analytics.jag [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<%
```
Comment (DilanUA) [inline modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/pages/device-analytics.jag:1]: are these implementations still in use?

**Comment by charithag** [comment]
This is no longer available in IoT Server.

File: modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/apis/stats-api.jag (hunk 1 above)
Comment (charithag) [inline modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/apis/stats-api.jag:1]: No

File: modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/app.js (hunk 2 above)
Comment (charithag) [inline modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/app.js:1]: No

File: modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/pages/device-analytics.jag (hunk 3 above)
Comment (charithag) [inline modules/distribution/src/repository/jaggeryapps/store/extensions/app/store-device/pages/device-analytics.jag:1]: No

**Pull request merged**